.github/workflows/ci.yml

```diff
@@ -2,17 +2,17 @@ name: Continuous Integration
 on:
   push:
     paths-ignore:
-      - "*.md"
+      - '*.md'
   pull_request:
     paths-ignore:
-      - "*.md"
+      - '*.md'
 
 jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [15]
+        node-version: [15, 16]
```

Comment:
do we still need to run it on Node 15?

Reply:
Engine in `package.json` is set at `>=15.0.0`, so... maybe?
I'm not keen on testing on non-LTS versions either, so if the general consensus is to get rid then I'm more than happy to remove.

Reply:
Removing the test version means we drop support of that node version. I think we should keep that.

Reply:
What do you think @kibertoad?

Reply:
`diagnostics_channel` was landed in v15. It might better to keep it.

Reply:
yeah, makes sense, better to keep it then